RedLockNet.SERedis.Shared/RedLock.cs

```diff
@@ -326,12 +328,13 @@ private void StartAutoExtendTimer()
 				(int) interval);
 		}
 
-		private long GetRemainingValidityTicks(long startTick)
+		private long GetRemainingValidityTicks(long startTimestamp)
 		{
 			// Add 2 milliseconds to the drift to account for Redis expires precision,
 			// which is 1 milliescond, plus 1 millisecond min drift for small TTLs.
 			var driftTicks = (long) (expiryTime.Ticks * clockDriftFactor) + TimeSpan.FromMilliseconds(2).Ticks;
-			var validityTicks = expiryTime.Ticks - (Stopwatch.GetTimestamp() - startTick) - driftTicks;
+		    var elapsedTicks = ConvertToTicks(Stopwatch.GetTimestamp() - startTimestamp);
```

Comment:
Could we use `Stopwatch` object and check `Elapsed.Ticks` here?

Reply:
Yes, I think that's a simpler approach. I'll make that change on the master branch. Thanks for the pull request nevertheless.